source/renderer/app/components/staking/delegation-center/WalletRow.js

```diff
@@ -178,160 +236,180 @@ export default class WalletRow extends Component<Props> {
-                        pendingDelegationStakePool &&
-                        !isNil(pendingDelegationStakePool.ranking)
-                          ? getColorFromRange(
-                              pendingDelegationStakePool.ranking,
-                              numberOfStakePools
-                            )
-                          : null;
-
-                      const tickerClasses = classnames([
-                        styles.ticker,
-                        isUnknown ? styles.unknown : null,
-                        'tickerText',
-                      ]);
-                      return [
-                        <PopOver
-                          key="ticker"
-                          content={
-                            <div className={styles.tooltipLabelWrapper}>
-                              <FormattedMessage
-                                {...messages.TooltipPoolTickerEpoch}
-                                values={{
-                                  fromEpoch,
-                                }}
-                              />
-                            </div>
-                          }
-                        >
-                          <span
-                            className={tickerClasses}
-                            style={{ color: pendingStakePoolColor }}
-                          >
-                            {pendingDelegation.target
-                              ? `[${
-                                  isUnknown
-                                    ? intl.formatMessage(
-                                        messages.unknownStakePoolLabel
-                                      )
-                                    : pendingDelegationStakePool.ticker
-                                }]`
-                              : notDelegatedText}
-                          </span>
-                        </PopOver>,
-                        !isLast && (
-                          <SVGInline
-                            key="arrow"
-                            svg={arrow}
-                            className={styles.arrow}
+                  </div>
+                ) : (
+                  <div className={styles.nonDelegatedText}>
+                    {notDelegatedText}
+                  </div>
+                )}
+              </div>
+              <SVGInline svg={arrow} className={styles.arrow} />
+              <div className={futureStakePoolTileStyles}>
+                {futurePendingDelegationStakePoolId ? (
+                  <div>
+                    {futurePendingDelegationStakePool ? (
+                      <PopOver
+                        key="stakePoolTooltip"
+                        placement="auto"
+                        maxWidth={300}
+                        popperOptions={{
+                          strategy: 'fixed',
+                        }}
+                        themeVariables={{
+                          '--rp-pop-over-bg-color':
```

Comment:
Can you define this variables object outside of render function and reference it here?

Reply:
Fixed